### ISSUE TITLE
AzResourceProvider doesn't accept Location

### ISSUE THREAD

**Issue by MarioRuiz** [Azure/azure-powershell#10946]
## Description
When calling `get-AzResourceProvider -ProviderNamespace "Microsoft.Netapp" -Location "xxxx" we get the message: 'A parameter cannot be found that matches parameter name 'Location'
When calling `get-help get-AzResourceProvider` Location is not an option on the output help.

## Steps to reproduce

```get-AzResourceProvider -ProviderNamespace "Microsoft.Netapp" -Location "xxxx"
get-help get-AzResourceProvider
```

## Environment data

```
Name                           Value
----                           -----
PSVersion                      6.2.3
PSEdition                      Core
GitCommitId                    6.2.3
OS                             Linux 4.18.0-25-generic #26-Ubuntu SMP Mon Jun 24 09:32:08 UTC …
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

```

## Module versions

```


```

## Debug output

```
    Directory: /home/rmario/.local/share/powershell/Modules

ModuleType Version    Name                                PSEdition ExportedCommands
---------- -------    ----                                --------- ----------------
Script     1.7.0      Az.Accounts                         Core,Desk {Disable-AzDataCollection,…
Script     1.6.5      Az.Accounts                         Core,Desk {Disable-AzDataCollection,…
Script     0.1.4      Az.NetAppFiles                      Core,Desk {Get-AzNetAppFilesAccount,…
Script     4.0.2      Az.Network                          Core,Desk {Get-AzApplicationGateway,…
Script     4.0.2      Az.Resources                        Core,Desk {Add-AzADApplicationOwner,…
Manifest   1.1.0.0    Microsoft.PowerShell.Archive        Desk      {Compress-Archive, Expand-…

    Directory: /snap/powershell/86/opt/powershell/Modules

ModuleType Version    Name                                PSEdition ExportedCommands
---------- -------    ----                                --------- ----------------
Manifest   1.2.3.0    Microsoft.PowerShell.Archive        Desk      {Compress-Archive, Expand-…
Manifest   6.1.0.0    Microsoft.PowerShell.Host           Core      {Start-Transcript, Stop-Tr…
Manifest   6.1.0.0    Microsoft.PowerShell.Management     Core      {Add-Content, Clear-Conten…
Manifest   6.1.0.0    Microsoft.PowerShell.Security       Core      {Get-Credential, Get-Execu…
Manifest   6.1.0.0    Microsoft.PowerShell.Utility        Core      {Export-Alias, Get-Alias, …
Script     1.3.2      PackageManagement                   Desk      {Find-Package, Get-Package…
Script     2.1.3      PowerShellGet                       Desk      {Find-Command, Find-DSCRes…
Script     0.0        PSDesiredStateConfiguration         Desk      {Get-ComplexResourceQualif…
Script     2.0.0      PSReadLine                          Desk      {Get-PSReadLineKeyHandler,…
Binary     1.1.2      ThreadJob                           Desk      Start-ThreadJob

```

## Error output

```
Message        : A parameter cannot be found that matches parameter name 'Location'.
StackTrace     :    at System.Management.Automation.CmdletParameterBinderController.VerifyArgum
                 entsProcessed(ParameterBindingException originalBindingException)
                    at System.Management.Automation.CmdletParameterBinderController.BindCommand
                 LineParametersNoValidation(Collection`1 arguments)
                    at System.Management.Automation.CmdletParameterBinderController.BindCommand
                 LineParameters(Collection`1 arguments)
                    at 
                 System.Management.Automation.CommandProcessor.BindCommandLineParameters()
                    at System.Management.Automation.CommandProcessor.Prepare(IDictionary 
                 psDefaultParameterValues)
                    at System.Management.Automation.CommandProcessorBase.DoPrepare(IDictionary 
                 psDefaultParameterValues)
                    at System.Management.Automation.Internal.PipelineProcessor.Start(Boolean 
                 incomingStream)
                    at System.Management.Automation.Internal.PipelineProcessor.SynchronousExecu
                 teEnumerate(Object input)
                 --- End of stack trace from previous location where exception was thrown ---
                    at System.Management.Automation.Internal.PipelineProcessor.SynchronousExecu
                 teEnumerate(Object input)
                    at System.Management.Automation.PipelineOps.InvokePipeline(Object input, 
                 Boolean ignoreInput, CommandParameterInternal[][] pipeElements, 
                 CommandBaseAst[] pipeElementAsts, CommandRedirection[][] commandRedirections, 
                 FunctionContext funcContext)
                    at System.Management.Automation.Interpreter.ActionCallInstruction`6.Run(Int
                 erpretedFrame frame)
                    at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction
                 .Run(InterpretedFrame frame)
Exception      : System.Management.Automation.ParameterBindingException
InvocationInfo : {Get-AzResourceProvider}
Line           : get-AzResourceProvider -ProviderNamespace "Microsoft.Netapp" -Location "xxxx"
Position       : At line:1 char:62
                 + ... sourceProvider -ProviderNamespace "Microsoft.Netapp" -Location "xxxx"
                 +                                                          ~~~~~~~~~
```

**Comment by dingmeng-xue** [comment]
@MarioRuiz , the input value of Location is pre-defined. Please try to use valid value such as westus, "West US"

**Comment by MarioRuiz** [comment]
Yes I was using a correct location

**Comment by markcowl** [comment]
@MarioRuiz It looks like you are using the preview version of Az.Resources, was that intentional?  This is supported in the stable version that is included with Az (1.10.0)

**Comment by MarioRuiz** [comment]
@markcowl I thought I was using a stable version. Reverting everything to stable it seems like solves the issue. Thanks
